pyproject.toml

```diff
@@ -47,13 +47,11 @@ packages = [
 [tool.poetry.dependencies]
 # Production dependencies.
 # Version syntax: https://python-poetry.org/docs/dependency-specification/
-python = "^3.6.0"
+python = ">=3.8.0"
 aiohttp = "^3"
-asgiref = "^3"
-backports-datetime-fromisoformat = {version = "^1", python = "~3.6"}
+asgiref = "^3.4.1"
 channels = "^3"
-dataclasses = {version = "^0.7", python = "~3.6"}
-Django = "^3"
+Django = "^4"
```

Comment:
should this not be a range to support Django 3.2 + 4.0? We've suffered from this previously

Reply:
Changed this to django `^3.1` (`>3.1 <4.0` and asgiref to "*".

It's not the busines of 

This lets the other dependencies resolve without friction and removes the upper bound.

Reply:
@tony doesn't this mean that Django 4.0 isn't supported, since it's `<4.0`? Perhaps we want `>=3.1,<4.1`?

Reply:
@eabruzzese that's an older ref but right now I have it at, `Django = "^3.1"`, which is also incorrect. I will loko at it

@matclayton Yes that'd be ideal

I will update it in case anyone wants to fork - but there's a bigger issue, I believe this project is abandoned by the maintainers.

Reply:
Updated, and I'm resolving this conversation (so the old value doesn't cause confusion)

If there is further pinning conversation / anything else make a note

And FYI once again it's likely this PR won't see the light of day. Maintainer has left :(